index.ts

```diff
@@ -18,14 +18,15 @@ const api = <T>(route: string, options?: Options): Promise<T> =>
     body: options?.body ? JSON.stringify(options.body) : undefined,
     headers: {
       ...options?.headers,
+      ["content-type"]: "application/json",
       Authorization: `api-key ${state.token}`,
     },
   }).then((r) => r.json());
 
-api.post = (route: string, body?: object, options?: ExtraOptions) =>
-  api(route, { method: "POST", body, ...options });
-api.patch = (route: string, body?: object, options?: ExtraOptions) =>
-  api(route, { method: "PATCH", body, ...options });
+api.post = <T>(route: string, body?: object, options?: ExtraOptions) =>
+  api<T>(route, { method: "POST", body, ...options });
+api.patch = <T>(route: string, body?: object, options?: ExtraOptions) =>
+  api<T>(route, { method: "PATCH", body, ...options });
```

Comment:
Just a minor TS thing here to make it a little easier to coerce the types. The types are lies, but they're convenient lies.